src/gridstack-poly.js

```diff
@@ -146,6 +146,23 @@ if (!Array.prototype.findIndex) {
   });
 })([Element.prototype, Document.prototype, DocumentFragment.prototype]);
 
+// from: https://github.com/jserz/js_piece/blob/master/DOM/Element/prepend()/prepend().md

```

Comment:
can't access that URL... looks like a private repo. isn't there a developer.mozilla.org version instead ?